### PR TITLE
sbc_sync_gp modul

### PR DESCRIPTION
**Pull request by Secheron**
Creation & configuration of the modul.

### CODE REVIEW

**Comment by Secheron** [comment]
Sync spoken languages with GP
================================

This module import all GP spoken languages from countries and partners
in odoo 8.0.
Spoken languages are iso 639-3

Partners spoken languages:
Select all spoken_langs with Partner CODEGA in GP (table `langueparlee`)
and import in odoo where partner_id is found

Countries spoken languages:
Select all spoken_langs with Country ISO3166 in GP (table `langueparlee`)
and import in odoo where country_id is found

All ids not found in odoo are displayed on a text field and stored in DB

During modul installation:
modul_sbc_sync_gp_during_installation

After modul installation
modul_sbc_sync_gp_after_installation2



Configuration
=============

To configure this module, you need to add settings in .conf file of Odoo:

* mysql_host = <mysql host of gp>
* mysql_db = <mysql database of gp>
* mysql_user = <mysql user of gp>
* mysql_pw = <mysql password of gp>
* smb_user = <samba user for pushing child pictures on the NAS>
* smb_pwd = <samba password>
* smb_ip = <IP Address of the NAS>
* smb_port = <Samba Port>
* gp_pictures = <Path to the folder of child pictures on the NAS>

File: sbc_sync_gp/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    'name': "SBC sync - Import spoken languages",
+    'version': '0.1',
+    'category': 'Other',
+    'summary': """
+        SBC - Import spoken_langs sync GP""",
+    'sequence': 150,
+    'author': 'Compassion CH',
+    'website': 'http://www.compassion.ch',
+    'description': """
+Sync spoken languages with GP
```
Comment (ecino) [inline sbc_sync_gp/__openerp__.py]: Put the description in a README.md file at the root of the module

Comment (Secheron) [inline sbc_sync_gp/__openerp__.py]: Added

**Comment by ecino** [comment]
Please fix pep8 errors (listed in travis build) :
./sbc_sync_gp/model/sbc_sync.py:16:1: E302 expected 2 blank lines, found 1
./sbc_sync_gp/model/sbc_sync.py:45:35: W291 trailing whitespace
./sbc_sync_gp/model/sbc_sync.py:63:21: E203 whitespace before ':'
./sbc_sync_gp/model/sbc_sync.py:84:35: W291 trailing whitespace
./sbc_sync_gp/model/sbc_sync.py:102:21: E203 whitespace before ':'
./sbc_sync_gp/model/sbc_sync.py:114:21: E203 whitespace before ':'

**Comment by Secheron** [comment]
Added

**Comment by ecino** [comment]
Travis still failed.

**Pull request closed without merging**